appdirs.py

```diff
@@ -395,7 +426,7 @@ def user_log_dir(appname=None, appauthor=None, version=None, opinion=True):
     """
     if system == "darwin":
         path = os.path.join(
-            os.path.expanduser('~/Library/Logs'),
+            os.path.expanduser('~{0}/Library/Logs'.format(_effective_user())),
             appname)
```

Comment:
Can you put the round bracket on a new line to match what you did in the other functions, or in the other functions don't put the round bracket on a new line.